apisix/plugins/prometheus/exporter.lua

```diff
@@ -80,9 +80,14 @@ local function extra_labels(name, ctx)
         for _, kv in ipairs(labels) do
             local val, v = next(kv)
             if ctx then
-                val = ctx.var[v:sub(2)]
-                if val == nil then
-                    val = ""
+                if v:find("$",1, true) == 1 then
```

Comment:
Would be better to use string.byte which is O(1).

Reply:
good advice & work done